### PR TITLE
invoice-generation-mail

### PR DESCRIPTION
**Pull request by Manish-Srans**
Sir, Please check it once. I have not removed any invoice model class fields.

### CODE REVIEW

File: src/main/java/com/srans/nestserver/model/Invoice.java [hunk 1]
```diff
@@ -1,200 +1,265 @@
...
 import javax.persistence.GenerationType;
-import javax.persistence.Id; 
+import javax.persistence.Id;
 import javax.persistence.Table;
 
-import org.hibernate.annotations.Formula;
 
 @Entity
 
 @Table(name = "invoice")
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/model/Invoice.java:13]: Do we need table for this?
I don't think we should save data, it's just a bean for placeholder, we no need to store it on database

File: src/main/java/com/srans/nestserver/controller/InvoiceController.java [hunk 2]
```diff
@@ -120,4 +135,73 @@ public Invoice createinvoice(@Valid @RequestBody Invoice invoice) {
...
+	@GetMapping(value = "/invoices/{ids}")
+	@ResponseBody
+	List<Invoice> getTenantById(@PathVariable long[] ids) {
+
+		List<Invoice> invoice = new ArrayList<>();
+
+		for (long id : ids) {
+			Invoice invoiceData = new Invoice();
+
+			System.out.println(id);
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/controller/InvoiceController.java:149]: Remove this system and use logger to log

File: src/main/java/com/srans/nestserver/controller/InvoiceController.java [hunk 3]
```diff
@@ -120,4 +135,73 @@ public Invoice createinvoice(@Valid @RequestBody Invoice invoice) {
...
+			List<Object> invoiceInfo = invoiceRepository.invoiceData(id);
+			for (Iterator<Object> iterator = invoiceInfo.iterator(); iterator.hasNext();) {
+				Object[] object = (Object[]) iterator.next();
+
+				for (int i = 0; i < object.length; i++) {
+
+					switch (i) {
+					case 0:
```
Comment (srans-admin) [inline src/main/java/com/srans/nestserver/controller/InvoiceController.java:157]: Is there in other way to catch the list , rather than switch?

**Review by srans-admin — CHANGES_REQUESTED** [review]
For now it's ok. I am not merging now, do pending changes and commit to same PR

**Review by srans-admin — CHANGES_REQUESTED** [review]
I did not see any other resolutions here

File: src/main/java/com/srans/nestserver/model/Invoice.java (hunk 1 above)
Comment (Manish-Srans) [reply src/main/java/com/srans/nestserver/model/Invoice.java:13]: Sir, you are right, but this table has been made by likhit and has also made some api, if you say I will delete

File: src/main/java/com/srans/nestserver/controller/InvoiceController.java (hunk 3 above)
Comment (Manish-Srans) [reply src/main/java/com/srans/nestserver/controller/InvoiceController.java:157]: I'll change switch to else if...

File: src/main/java/com/srans/nestserver/controller/InvoiceController.java (hunk 2 above)
Comment (Manish-Srans) [reply src/main/java/com/srans/nestserver/controller/InvoiceController.java:149]: Sir, I don't understand, You mean adding logger.

**Pull request merged**